benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -222,6 +223,37 @@ class KokkosScopeGuard
       ->UseManualTime()                                                        \
       ->Unit(benchmark::kMicrosecond);
 
+#else
+#define REGISTER_BENCHMARK(TreeType, r1, r2, r3)                               \
```

Comment:
Just remove r1-r3, and hardcode. I see no point in trying to be general. This will be gone soon (fingers crossed).